cformers/interface.py

```diff
@@ -1,16 +1,13 @@
 """Call's the C++ code from Python."""
 from subprocess import Popen, PIPE
 import hashlib
-import re
 import os
 import sys
-import select
-import wget
```

Comment:
It seems like this has not been updated against the latest push to `master` and is reverting a lot of the changes made
Could you please address this?

Reply:
Done